### PR TITLE
Add red badge to reports tab in Android

### LINKED ISSUE #7720: Add a red badge to the Reports tab
## 🎯 What needs to be done?
At the same time that we show the final "You've made your first time entry" tooltip, we should also add a 🔴 badge to the reports tab to communicate that something has changed there.

## 🎨 Graphical resources
https://toggl.invisionapp.com/share/F6YFHZW8SZH#/screens/428592016

## 🏆 Outcome
* The badge shows up at the same time as the final tooltio
* It disappears after the user goes to the reports tab

### PR DESCRIPTION
**Pull request by arturs105**
## What's this?
Adds a red badge to reports tab once onboarding is finished

### Relationships

Closes #7720

## :squid: Permissions
🦑

### CODE REVIEW

**Comment by daividssilverio** [comment]
Unrelated, but I got this while testing this run. We might want an issue for that. Not sure if we should show the tooltip bellow the play button or show it at all if we don't have the space for it.
Screen Shot 2020-09-21 at 11 50 07 AM

File: Toggl.Droid/Activities/MainTabBarActivity.cs [hunk 1]
```diff
@@ -166,7 +166,10 @@ private async Task<Fragment> getCachedFragment(int itemId)
                 {
                     var viewModel = ViewModel.MainViewModel.Value as MainViewModel;
                     await viewModel.Initialize();
-                    return new MainFragment { ViewModel = viewModel };
+                    return new MainFragment(() =>
+                    {
+                        navigationView.GetOrCreateBadge(navigationView.Menu.GetItem(1).ItemId);
```
Comment (daividssilverio) [inline Toggl.Droid/Activities/MainTabBarActivity.cs:171]:
```suggestion
                        navigationView.GetOrCreateBadge(Resource.Id.MainTabReportsItem);
```

File: Toggl.Droid/Activities/MainTabBarActivity.cs [hunk 2]
```diff
@@ -224,11 +227,16 @@ private async Task showFragment(int fragmentId)
...
                 return;
-            
+
             SupportFragmentManager.ExecutePendingTransactions();
             var transaction = SupportFragmentManager.BeginTransaction();
             var fragment = await getCachedFragment(fragmentId);
 
+            if (fragment is ReportsFragment)
+            {
+                navigationView.RemoveBadge(navigationView.Menu.GetItem(1).ItemId);
```
Comment (daividssilverio) [inline Toggl.Droid/Activities/MainTabBarActivity.cs:237]:
```suggestion
                navigationView.RemoveBadge(Resource.Id.MainTabReportsItem);
```

**Review by daividssilverio — CHANGES_REQUESTED** [review]
Tiny suggestion.
Code looks and works fine already.
Found a small visual issue while testing. The reports adapter also crashes with the same issue I've mentioned [here](https://github.com/toggl/mobileapp/pull/7772#discussion_r492160699).

**Comment by arturs105** [comment]
@daividssilverio I know about the reports adapter crashing - it's because i added some new `ReportSomethingElement` for the empty states during iOS development, but i didn't add Android UI counterparts (view holders & layouts). I'm already working on those, so a PR that fixes the crashes will be ready shortly.

**Review by daividssilverio — APPROVED** [review]

**Pull request merged**